### PR TITLE
plugins: Add Youtube-HLS to the Service dropdown

### PR DESCRIPTION
**Pull request by mv2731**
This commit changes the name of the option called Youtube
/ Youtube Gaming to Youtube-RTMP to be more specific
and adds a Youtube-HLS option in the Stream settings
Service dropdown.

### CODE REVIEW

File: plugins/rtmp-services/data/package.json [hunk 1]
```diff
@@ -1,10 +1,5 @@
 {
-	"url": "https://obsproject.com/obs2_update/rtmp-services",
-	"version": 132,
-	"files": [
-		{
-			"name": "services.json",
-			"version": 132
-		}
-	]
+	"url" : "https://obsproject.com/obs2_update/rtmp-services",
```
Comment (ushadow) [inline plugins/rtmp-services/data/package.json]: Is the formatting change produced by clang-format?

Comment (mv2731) [reply plugins/rtmp-services/data/package.json]: I ran clang-format -i package.json on the file, so probably? but I can check...also when I run this I see
image

File: plugins/rtmp-services/data/services.json [hunk 2]
```diff
@@ -198,7 +198,27 @@
...
+            "name": "YouTube-HLS",
+            "common": true,
+            "servers": [
+                {
+                    "name": "Primary YouTube ingest server",
+                    "url": "https://a.upload.youtube.com/http_upload_hls?cid=<stream key>&copy=0&file=out.m3u8"
+                },
+                {
+                    "name": "Backup YouTube ingest server",
+                    "url": "https://b.upload.youtube.com/http_upload_hls?cid=<stream key>&copy=0&file=out.m3u8"
```
Comment (ushadow) [inline plugins/rtmp-services/data/services.json]: copy=1

File: plugins/rtmp-services/data/services.json [hunk 3]
```diff
@@ -198,7 +198,27 @@
             }
         },
         {
-            "name": "YouTube / YouTube Gaming",
+            "name": "YouTube-HLS",
+            "common": true,
+            "servers": [
+                {
+                    "name": "Primary YouTube ingest server",
+                    "url": "https://a.upload.youtube.com/http_upload_hls?cid=<stream key>&copy=0&file=out.m3u8"
```
Comment (ushadow) [inline plugins/rtmp-services/data/services.json]: cid={stream_key}

And same below.

**Pull request merged**